platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleStateRoot.java

```diff
@@ -198,11 +210,7 @@ public MerkleStateRoot() {
      * on restart, on reconnect, or any other time indicated by the {@code trigger}.
      */
     @Override
-    public void init(
-            final Platform platform,
-            final PlatformState platformState,
-            final InitTrigger trigger,
-            final SoftwareVersion deserializedVersion) {
+    public void init(final Platform platform, final InitTrigger trigger, final SoftwareVersion deserializedVersion) {
```

Comment:
I see in many places, previously multi-lined arg list are now inlined. Not asking to change it, but we probably want to keep it consistent. A 3-args list should probably remain multi-lined.

Reply:
I basically let Spottles to decide how it should look like but I see your point. Let me update it